client/components/field-error/index.js

```diff
@@ -1,17 +1,22 @@
 import React, { PropTypes } from 'react';
 import sanitizeHTML from 'lib/utils/sanitize-html';
 import Gridicon from 'gridicons';
+import classNames from 'classnames';
 
-const FieldError = ( { text } ) => {
+const FieldError = ( { text, type = 'input-validation' } ) => {
 	return (
-		<div className="form-input-validation is-error">
+		<div className={ classNames(
+			'is-error',
+			[ `form-${ type }` ]
```

Comment:
> Should we be hard-coding the form prefix to the CSS class? What if I want to use some other CSS class for the div?

What's your imagined use case?

The `Field` in the `FieldError` component references a `<form>` field.